### PR TITLE
Add an action to restore /ect/dovecot/dovecto.conf to state before conversion/distupgrade

### PR DESCRIPTION
**Pull request by SandakovMM**

### CODE REVIEW

File: pleskdistup/actions/emails.py [hunk 1]
```diff
@@ -52,3 +54,31 @@ def _revert_action(self) -> action.ActionResult:
...
+
+    def _prepare_action(self) -> action.ActionResult:
+        files.backup_file(self.dovecot_config_path)
+        return action.ActionResult()
+
+    def _post_action(self) -> action.ActionResult:
+        path_to_backup = plesk.CONVERTER_TEMP_DIRECTORY + "/dovecot.conf.bak"
+        if os.path.exists(self.dovecot_config_path):
+            shutil.copy(self.dovecot_config_path, path_to_backup)
+            motd.add_finish_ssh_login_message(f"The dovecot configuration '{self.dovecot_config_path}' has been restored from CentOS 7. Modern configuration was placed in '{path_to_backup}'.")
```
Comment (kpushkaryov) [inline pleskdistup/actions/emails.py:77]: Does anything confines this action to CentOS 7 only? If not, propose to drop CentOS 7 mention from the message.

File: pleskdistup/actions/emails.py [hunk 2]
```diff
@@ -52,3 +54,31 @@ def _revert_action(self) -> action.ActionResult:
...
+
+    def _is_required(self) -> bool:
+        return os.path.exists(self.dovecot_config_path)
+
+    def _prepare_action(self) -> action.ActionResult:
+        files.backup_file(self.dovecot_config_path)
+        return action.ActionResult()
+
+    def _post_action(self) -> action.ActionResult:
+        path_to_backup = plesk.CONVERTER_TEMP_DIRECTORY + "/dovecot.conf.bak"
```
Comment (kpushkaryov) [inline pleskdistup/actions/emails.py:74]: What is `plesk.CONVERTER_TEMP_DIRECTORY`? Can't find it's definition.
Also, please use `os.path.join()`.

**Review by kpushkaryov — CHANGES_REQUESTED** [review]
Also, typos in the commit message: "ect", "dovecto.conf".

File: pleskdistup/actions/emails.py (hunk 1 above)
Comment (SandakovMM) [reply pleskdistup/actions/emails.py:77]: I think could be used for other repositories as well, so lets remove mention of centos 7

File: pleskdistup/actions/emails.py (hunk 2 above)
Comment (SandakovMM) [reply pleskdistup/actions/emails.py]: > What is plesk.CONVERTER_TEMP_DIRECTORY?

It should be passed as argument, actually it is state_dir.

**Comment by kpushkaryov** [comment]
Please fix commit message typos before merge.

**Review by kpushkaryov — APPROVED** [review]

**Pull request merged**